### DISCUSSION TITLE
Exporting diagram to svg

### DISCUSSION THREAD

**Discussion by PatrikN-work** [eclipse-glsp/glsp#1016] [Q&A] [answered]
Hi,

Is there a simple way to export diagram as svg files to include in e.g. documentation?

**Comment by tortmayr** [comment, marked as answer]
HI @PatrikN-work,
Yes GLSP provides support for exporting Diagrams as SVG out of the box.
The correct implementation is dependent on the chosen application frame (Theia, VS Code, standalone) but
typically all integrations at least offer the possibility to export the diagram via
- Menu (`Diagram` -> `Export as SVG`)
- Keyboard shortcut (`Alt+e`)

If I'm not mistaken you are currently looking at the `node-json-vscode` project template. So here is a short demo with this example:


![export-demo](https://github.com/eclipse-glsp/glsp/assets/2311075/3430773b-c08a-4c14-8fa2-c74242d68ced)

(Note: In older versions of the project template the diagram menu was misconfigured and not visible. But this has been fixed recently and should be available with the latest master.)

**Reply by PatrikN-work** [reply]
However, I can't make it work in Theia
